ee_tests/src/page_objects/login.page.ts

```diff
@@ -47,10 +47,14 @@ export class LoginPage extends BasePage {
     await this.passwordInput.enterText(password);
     await this.loginButton.clickWhenReady();
 
+    support.writeScreenshot('target/screenshots/login_1.png');
```

Comment:
It doesn't make sense to take a screenshot immediately after clicking the button... the page will be always in transition state at this point. Move it before `await this.loginButton.clickWhenReady()`

Reply:
Done